lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -1337,15 +1508,15 @@ def test_lazydata_maskedints(self):
         self.assertEqual(cube.dtype, np.dtype('i4'))
         self.assertEqual(cube.core_data.dtype, np.dtype('f8'))
         data = cube.data
-        self.assertTrue(np.ma.is_masked(data))
+        self.assertTrue(ma.is_masked(data))
         self.assertMaskedArrayEqual(data, masked_data_original)
 
     def test_lazydata_floating_dtype_change(self):
         # Check that re-assigning dtype won't allow a floating type.
         cube = self._sample_cube(dtype=np.float32, lazy=True,
                                  cube_fill_value=23.7)
         self.assertArrayAllClose(cube.fill_value, 23.7)
-        msg = "Can only cast lazy data to integral dtype"
+        msg = "Can only cast lazy data to integral or bool dtype"
```

Comment:
I think "integer" might now be clearer here. You might also want to change "bool" to "boolean".